src/components/Sidebar/components/CloseButton/CloseButton.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled, { css } from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import CloseIcon from './closeIcon.svg';
+
+const baseStyles = ({ theme, visible }) => css`
+  label: close-button;
+  outline: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  background-color: ${theme.colors.n100};
+  position: absolute;
+  bottom: ${theme.spacings.mega};
+  right: ${theme.spacings.mega};
+  transition: opacity 150ms ease-in-out, visibility 150ms ease-in-out;
+  visibility: ${visible ? 'visible' : 'hidden'};
+  opacity: ${visible ? 1 : 0};
+  z-index: 1;
+  ${theme.mq.mega`
+    visibility: hidden;  
+  `};
+`;
+
+const FloatingButton = styled('button')(baseStyles);
+
+const CloseButton = ({ visible, onClick }) => (
```

Comment:
This button lacks a text label to make it accessible to screen readers. You can either add it as a `span` and [hide it visually](https://polished.js.org/docs/#hidevisually) or you use the [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute. The first method is better supported. The label text should be a prop so that it can be translated.

Reply:
Do you mean to add a `label` (what do you think is a good name for it?) prop to the `Sidebar` component with the text label that would be added to the (hidden) `span` on `CloseButton`?

Thanks a LOT for the accessibility hints! 😄

Reply:
Yeah, that's what I mean. How about `closeLabel` or `closeButtonLabel` to be explicit?